src/Application/Cli/PreviewCommand.php

```diff
@@ -78,10 +77,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (!$this->isFound($input->getOption('livereload'))) {
-            $output->writeln('<error>Impossible to launch Livereload, did you forgot to install it with "pip install livereload" (sudo maybe required)?</error>');
+        if ($input->getOption('livereload')) {
+            if ($input->getOption('livereload') === null) {
+                $input->setOption('livereload', 'livereload');
+            }
```

Comment:
I don't think it's possible to go into that `if` since we did:

    if ($input->getOption('livereload')) {

just above (so `$input->getOption('livereload')` cannot be null)